src/coreclr/interpreter/compiler.cpp

```diff
@@ -1364,7 +1364,7 @@ void InterpCompiler::EmitBinaryArithmeticOp(int32_t opBase)
     }
     else
     {
-#if SIZEOF_VOID_P == 8
+#if __SIZEOF_POINTER__ == 8
```

Comment:
nit: we don't have its precedence in coreclr partition, better: `#ifdef TARGET_64BIT` or `#if TARGET_POINTER_SIZE == 8`.